src/Node.fs

```diff
@@ -2092,7 +2098,7 @@ module net =
         abstract Socket: SocketStatic
         abstract Server: ServerStatic
         abstract createServer: ?connectionListener: (Socket -> unit) -> Server
-        abstract createServer: ?options: obj * ?connectionListener: (Socket -> unit) -> Server
+        abstract createServer: ?options: CreateServerOptions * ?connectionListener: (Socket -> unit) -> Server
```

Comment:
I'm leaning towards `CpuInfoCreateServerOptions` instead of `CreateServerOptions`.